### DISCUSSION TITLE
How to set up for a podcast?

### DISCUSSION THREAD

**Discussion by ScottBurger** [daattali/beautiful-jekyll#1026] [Q&A] [answered]
Is there a guide to set up a post for automatically updating a podcast RSS? If I set a `file: https://filename.mp3` in the _post yaml, what else would I need to do to have the mp3 file show up in the feed.xml file? Is it as simple as just adding a section in the feed.xml for the `file` key in the yaml of the post?

**Comment by ScottBurger** [comment, marked as answer]
So according to [Apple's documentation](https://help.apple.com/itc/podcasts_connect/#/itcb54353390), there are some required fields needed in an .xml file for it to be recognized as a podcast vs a normal blog .xml feed. 

The minimum required are:

- title
- description
- itunes:image
- language
- itunes:category
- itunes:explicit

You'll also need the `xmlns:itunes="http://www.itunes.com/dtds/podcast-1.0.dtd"` bit in the `rss version` tag to have the itunes-specific xml tags be included properly.

[I have a working example here](https://github.com/ScottBurger/GROGtest2/blob/master/feed.xml) that illustrates what fields to add and where.

**Reply by daattali** [reply]
Thanks for coming back here to provide the answer to others!
